src/extras/global.js

```diff
@@ -10,18 +10,21 @@
   var firstGlobalProp, secondGlobalProp, lastGlobalProp;
   function getGlobalProp () {
     var cnt = 0;
-    var lastProp;
+    var foundLastProp, result;
     for (var p in global) {
       // do not check frames cause it could be removed during import
       if (shouldSkipProperty(p))
         continue;
       if (cnt === 0 && p !== firstGlobalProp || cnt === 1 && p !== secondGlobalProp)
         return p;
+      if (foundLastProp && !shouldSkipProperty(p)) {
+        lastGlobalProp = p;
+        result = result || p;
```

Comment:
As per feedback, unless you want to wait on the next major to release this perhaps worth adding back -

```suggestion
        result = System.firstGlobalProp && result || p;
```

Reply:
Ah okay, I misunderstood and thought the next major would be happening soon. I'll add this back.